steve/util.py

```diff
@@ -155,17 +222,115 @@ def get(self, section, key):
     # api_key =
     """)
 
-    def create(self, dir_path):
-        abs_path = os.path.abspath(dir_path)
-        kw = dict(
-            projectpath=abs_path,
-            jsonpath=os.path.join(abs_path, 'json')
-        )
-        with open(os.path.join(dir_path, self.file_name), 'w') as fp:
-            fp.write(self._CONFIG.format(**kw))
 
+class YamlConfig(Config):
+    file_name = 'steve.yaml'
+
+    def __init__(self):
+        super(YamlConfig, self).__init__()
+
+    def load(self):
+        """Finds and opens the config file in the current directory
+
+        :raises ConfigNotFound: if the config file can't be found
+
+        :returns: config file
+
+        """
+        self._data = yaml_load(open(self.project_file))
+        project = self._data['project']
 
-SteveConfig = IniConfig   # to allow easy switching in the future
+        # TODO: This is a little dirty since we're inserting stuff into
+        # the config file if it's not there, but so it goes.
+        try:
+            project['projectpath']
+        except KeyError:
+            project['projectpath'] = self.project_path
+        try:
+            project['jsonpath']
+        except KeyError:
+            project['jsonpath'] = os.path.join(project['projectpath'], 'json')
+        try:
+            project['yamlpath']
+        except KeyError:
+            project['yamlpath'] = os.path.join(project['projectpath'], 'yaml')
```

Comment:
Would a project ever have json files and yaml files? Maybe we should change this to just `data` and have the file endings denote which kind of file they are?